data/gegno/gegno fleets.txt

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
```suggestion
```
I assume this is a mistake; in all other files there isnt an empty space before the copyright line